import PureCloudPlatformClientV2
from PureCloudPlatformClientV2.rest import ApiException
from pprint import pprint

# Configure OAuth2 access token for authorization: PureCloud OAuth
PureCloudPlatformClientV2.configuration.access_token = 'your_access_token'
# or use get_client_credentials_token(...), get_saml2bearer_token(...) or get_code_authorization_token(...)

# create an instance of the API class
api_instance = PureCloudPlatformClientV2.ObjectsApi();
division_id = 'division_id_example' # str | Division ID
body = PureCloudPlatformClientV2.AuthzDivision() # AuthzDivision | Recreated division data

try:
    # Recreate a previously deleted division.
    api_response = api_instance.post_authorization_division_restore(division_id, body)
    pprint(api_response)
except ApiException as e:
    print("Exception when calling ObjectsApi->post_authorization_division_restore: %s\n" % e)